src/ripple/rpc/tests/JSONRPC.test.cpp

```diff
@@ -1112,6 +1112,45 @@ R"({
 "Missing field 'tx_json.TransactionType'.",
 "Missing field 'tx_json.TransactionType'."}},
 
+{ "Invalid field 'tx_json': not an object",
```

Comment:
Suggestion: Change comment to "Invalid field 'tx_json': string" or change tx_json value to "null".